### ISSUE TITLE
No audio when adding Mp3 to VideoFileClip MoviePy

### ISSUE THREAD

**Issue by dkhonig** [Zulko/moviepy#350]
I'm trying to add an mp3 audio file to a video clip that I'm creating out of images with MoviePy. When the script runs it creates the mp4 file and plays successfully, however there's no audio. I'm not really sure why and can't seem to find a ton of documentation around this in general. MoviePy is pretty new to me so any help would be appreciated - thank-you!

Asked on stackoverflow: http://stackoverflow.com/questions/40445885/no-audio-when-adding-mp3-to-videofileclip-moviepy

```
def make_video(images):
    image_clips = []
    for img in images:
        if not os.path.exists(img):
            raise FileNotFoundError(img)
        ic = ImageClip(img).set_duration(3)
        image_clips.append(ic)

    video = concatenate(image_clips, method="compose")
    video.set_audio(AudioFileClip("audio.mp3")) 
    video.write_videofile("mp4_with_audio.mp4", fps=60, codec="mpeg4")
```

**Comment by kerstin** [comment]
@dkhonig Based on the SO comments, it looks like you managed to solve this? Is this something that could/should be better documented? The project accepts pull requests in case you'd like to help make the docs better yourself!

**Comment by dkhonig** [comment]
Yes I did. If I remember correctly - It was really only an issue for my specific use case although, it might be helpful to outline it in the docs. The issue was that when I was generating an audio clip, the duration was not matching up with the ImageClip's duration resulting in no audio. Once I properly understood what I was doing, I was able to see that the durations for everything needed to be more precise. I was chopping up audio files and creating custom video's with clip's of audio files and still images that I created. The duration of all of the image clips, audio clip and fps needed to all be insync and fine tuned.
